gdal/gcore/gdal_priv.h

```diff
@@ -813,6 +813,7 @@ class CPL_DLL GDALDataset : public GDALMajorObject
                                     OGRGeometry *poSpatialFilter,
                                     const char *pszDialect,
                                     swq_select_parse_options* poSelectParseOptions);
+    virtual OGRErr      AbortSQL( );
```

Comment:
must be moved close below ReleaseResultSet() at line 793. Here it is a section with doxygen suppression